yt_dlp/extractor/wasdtv.py

```diff
@@ -0,0 +1,176 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from yt_dlp.extractor.common import InfoExtractor
+from yt_dlp.compat import (
+    compat_kwargs,
+    compat_str,
+)
+from yt_dlp.utils import (
+    ExtractorError,
+    int_or_none,
+    parse_iso8601,
+    urljoin,
+    try_get,
+)
+
+
+class WASDTVBaseIE(InfoExtractor):
+    _API_BASE = 'https://wasd.tv/api/'
+    _THUMBNAIL_SIZES = ('small', 'medium', 'large')
+
+    def _fetch(self, *args, **kwargs):
```

Comment:
why is this needed?
u are not even using fata=false
the code will never show ur ExtractorError message.

Reply:
I see that u are using it to clean up the code.
Yeah it's nice to have this then.
Just hardcode the args u are consuming and are def needed and rest can be left to kwargs and args